e2e-tests/security-context/compare/statefulset_sec-context-rs0-changed-oc.yml

```diff
@@ -0,0 +1,225 @@
+apiVersion: apps/v1
```

Comment:
why this file needed if it is identical to `e2e-tests/security-context/compare/statefulset_sec-context-rs0-changed-oc.yml`

Reply:
removed